src/dlgConnectionProfiles.h

```diff
@@ -71,16 +71,19 @@ public slots:
     bool validateConnect();
     void updateDiscordStatus();
     bool validateProfile();
+    void removeNotification(QString notif);
+    void addNotification(QString notif);
 
-    bool validName;
-    bool validUrl;
-    bool validPort;
 
+    bool validProfile;
     QStringList mProfileList;
-    QPalette mRegularPalette;
-    QPalette mOKPalette;
-    QPalette mErrorPalette;
-    QPalette mReadOnlyPalette;
+    QStringList mErrorList;
+
+    QString mStyleSheetRegular= "* { color:#0000c0; background: #0000C0; }";
```

Comment:
**I18n** These four should be `QStringLiteral`s or `QLatin1String`s...

Also, it is probably wise to target the specific widget(s) classes e.g. `.QLineEdit` rather than generally `*` so that the colour does not leak into other aspects (e.g. tooltips).